src/pages/register-a-beacon/check-your-answers.tsx

```diff
@@ -369,6 +413,14 @@ const CommunicationsSubSection: FunctionComponent<CheckYourAnswersBeaconUseSecti
           ) : (
             ""
           )}
+          {use.otherCommunication ? (
```

Comment:
Can use `&&` for consistency

Reply:
See other use fields above as well that can be updated to use `&&` over a ternary operator